crypto/types.go

```diff
@@ -92,5 +98,7 @@ const (
 	maxRelicPrgSeed = 1 << 32
 )
 
+const ()
```

Comment:
What's this for?

Reply:
This was not cleaned either, thanks for pointing it out.